api/tests/unit/domain/services/certification/certification-result-service_test.js

```diff
@@ -1225,4 +1226,75 @@ describe('Unit | Service | Certification Result Service', function() {
 
   });
 
+  describe('#computeAnswersSuccessRate', () => {
+
+    context('when no answers is given', () => {
+
+      it('should have a trust level has unknown', () => {
+        // given
+        const answers = [];
+
+        // when
+        const successRate = certificationResultService._computeAnswersSuccessRate(answers);
+
+        // then
+        expect(successRate).to.equal(0);
+      });
+    });
+
+    context('when all answers are OK', () => {
+
+      it('should have a success rate of 100%', () => {
+        // given
+        const answers = [new Answer({ result: 'ok' }), new Answer({ result: 'ok' })];
+
+        // when
+        const successRate = certificationResultService._computeAnswersSuccessRate(answers);
+
+        // then
+        expect(successRate).to.equal(100);
+      });
+    });
+
+    context('when all answers are KO', () => {
+
+      it('should have a success rate of 0%', () => {
+        // given
+        const answers = [new Answer({ result: 'ko' }), new Answer({ result: 'ko' })];
+
+        // when
+        const successRate = certificationResultService._computeAnswersSuccessRate(answers);
+
+        // then
+        expect(successRate).to.equal(0);
+      });
+    });
+
+    context('when the answers are a mixed of valid and wrong answers', () => {
+
+      it('should have a success rate of 50% with 1W and 1R', () => {
```

Comment:
Pourquoi pas 1 right and one wrong ? 
Ou mieux, pour rappeler Answer, 1 ok and 1 ko  (peut être trompeur) ?